### ISSUE TITLE
Balancing issues on masks

### ISSUE THREAD

**Issue by Tri-eDge17** [JoshTClark/RoR2Mod#6]
Just want to preface, I love the mod. Been using it a lot, but the masks, they die too quickly. Then its completely useless. They don't even respawn on next stages. I'd try to exchange them as fast as possible because they will die and be a dead item in sometimes like 10 seconds in a typhoon run. Its a bit useless at this current state.

Maybe make them respawn again like beetle guards instead of being dead items?

**Comment by JoshTClark** [comment]
I kinda want to stay away from making them respawn as the idea behind the mask items was each time you pick one up and it dies all future wisps get even stronger. Do you think a buff to the wisps in general would make the items more worthwhile? Maybe giving them a little regen, more health, or damage? I might also need to buff the effects the broken masks give as I definitely think the broken lesser mask is too weak as I think it's only a 25% HP increase. I was also thinking of toning down wisp damage by a lot but giving the a lot more HP and attack speed so they feel more consistent.

**Comment by Tri-eDge17** [comment]
I've been playing these in typhoon exclusively and I can tell you this, there are a ton of common items even in just vanilla that do a lot more than the masks right now. Quick examples that might be a good parallel in common are delicate watches which will give you a 20% boost in damage that will break when low health. Delicate watches give 20% boost on all watches and you can manage the threshold because you as a player has agency on how to handle being damaged. Wisps on the otherhand, will die in the next 10 seconds sometimes less if you cannot exchange them immediately, and after breaking, they're dead items on your end. Its a big detriment early in runs when you want to be more frugal with spending but in later runs they would die more immediate as well. They have less health than drones and are more prone to die easier as well in my experience. And stacking masks on masks on masks and spending your scrap on them if ever you find a printer doesnt seem worth either because they will just break anyway when it dies and there goes all your scrap. Wisps will never be your main way of dealing damage anyway because it doesn't inherit items which will be your main source of damage later in the runs. I really feel like you need to make them respawn to make them feel worth to some extent, (at least to divert some damage/aggression). The getting stronger each time wouldn't really even feel like its op because once its broken, how lucky do you have to be to get another mask? or a printer?

Anyway, hope you take it in consideration. Cheers!

**Comment by Darkmega18** [comment]
I second these also. I wanna play the wisps, but I also noticed how quick they die, even on more average difficulties when I just casually pick one up.

Could the wisp respawn on a cooldown or maybe you spawn allied wisps after killing elites (with a chance?) and instead the wisp strength, max number and chance of spawn is based on number of masks?

if anything ranged so much as looks at the poor wisp it just dies and becomes a mostly dud item.

if you must go the breaking summon route, the broken mask definitely needs a buff. and perhaps each mask breaks one by one (if theres a stack). so each subsequent one is spawned after, stronger.
something interesting could be maybe letting the wisp have all it's stats sped up by the broken mask?
HP 50%, attack 50%, regen, a couple a second? move speed? maybe even a touch of armor? with higher tier masks providing more buffs, since even they don't come back.
or have a cap on masks that need to break where each mask gives the wisp a chance to death deny and just respawn next to you instead of consuming the mask when it goes down. where on the 5th it's just an endlessly respawning follower. Much like one other mod that does an elite item that summons and constantly resummons the prototype drone as a legendary item.

**Comment by aaronfreed** [comment]
Yeah, I feel like these are currently just an instant scrap. The idea is nice, but the stat bonus currently isn't enough to make it worthwhile. Several of the ideas above seem a lot better. If it has to break the mask on the wisp's death, it should inherit your items or something to make it less likely to just die immediately.
